app/scripts/migrations/030.js

```diff
@@ -0,0 +1,22 @@
+const clone = require('clone')
+
+const version = 30
```

Comment:
It's worth noting that #5843 is concurrently introducing migration 30

cc @danjm